### PR TITLE
Adds the original tags to the new instance on the configure_instance …

### PR DESCRIPTION
**Pull request by Cesarla**
…step

### CODE REVIEW

**Comment by slitsche** [comment]
👍

File: planb/update_cluster.py [hunk 1]
```diff
@@ -325,7 +326,10 @@ def configure_instance(ec2: object, volume: dict, options: dict):
         return
 
     instance_id = instance['InstanceId']
-    create_tags(ec2, instance_id, {'Name': options['cluster_name']})
+    create_tags(ec2, instance_id, {
```
Comment (a1exsh) [inline planb/update_cluster.py]: Actually, what our convenience function `create_tags` does, is mapping the tags dict to a list of `{Key:,Value:}` dicts expected by the boto3 function.  In this case it is more straightforward to call the latter one directly: `ec2.create_tags(Resources=[instance_id], Tags=saved_instance['Tags'])`

**Comment by Cesarla** [comment]
@a1exsh Comments addressed

**Comment by a1exsh** [comment]
:+1:

**Comment by slitsche** [comment]
👍

**Pull request merged**